main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"github.com/fogo-sh/gutenbuild/runtime"
+
+	"github.com/BurntSushi/toml"
+
+	"os"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+
+	"io/ioutil"
+)
+
+func main() {
+	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
+	log.Logger = zerolog.New(os.Stdout)
+
+	log.Print("starting gutenbuild")
+
+	pipeline_content_bytes, err := ioutil.ReadFile("./pipeline.toml")
+	if err != nil {
+		log.Error().Err(err).Msg("error reading pipeline.toml")
+	}
+
+	pipeline_content := string(pipeline_content_bytes)
+
+	var pipeline runtime.Pipeline
```

Comment:
Since we aren't using var anywhere else, can this use `new` to create the variable to hold the pipeline? I _believe_ this should work fine with the toml loader

```suggestion
	pipeline := new(runtime.Pipeline)
```